sw/airborne/modules/sensors/airspeed_sdp3x.c

```diff
@@ -35,6 +36,23 @@
 #include "subsystems/datalink/telemetry.h"
 #endif
 
+#ifndef USE_AIRSPEED_SDP3X
+#if USE_AIRSPEED
+#define USE_AIRSPEED_SDP3X TRUE
+PRINT_CONFIG_MSG("USE_AIRSPEED_SDP3X set to TRUE since this is set USE_AIRSPEED")
+#endif
+#endif
+
+#if USE_AIRSPEED_SDP3X
+#include "state.h"
```

Comment:
At the moment, most airspeed sensor drivers are setting directly the state interface, which is not really nice. I would prefer that sensor just send an ABI message (either AIRSPEED or BARO_DIFF depending on the sensor) and the air_data module set the state interface in the end.